### PR TITLE
Release 0.9b1

### PR DESCRIPTION
**Pull request by tatiana**
We'll validate this release and make sure we merge the updated docs before releasing 0.9b.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.9.0b
```
Comment (kaxil) [inline docs/CHANGELOG.md:3]:
```suggestion
## 0.9.0b1
```

File: src/astro/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """A decorator that allows users to run SQL queries natively in Airflow."""
 
-__version__ = "0.8.2"
+__version__ = "0.9.0b"
```
Comment (kaxil) [inline src/astro/__init__.py:3]:
```suggestion
__version__ = "0.9.0b1"
```

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
+* The only mandatory property of a `Table` instance is `conn_id`. If no metadata is given, the library will try to extract schema and other information from the connection object. If it is missing, it will default to the `AIRFLOW__ASTRO__SQL_SCHEMA` environment variable.
+
+Internals:
+* Major refactor introducing `Database`, `File`, `FileType` and `FileLocation` concepts.
+
 ## 0.8.4
 
 Enhancements:
-* Add support for Airflow 2.3 [#329](https://github.com/astronomer/astro-sdk/pull/367).
+* Add support for Airflow 2.3 [#367](https://github.com/astronomer/astro-sdk/pull/367).
```
Comment (kaxil) [inline docs/CHANGELOG.md:22]: I had a typo in the PR I had merged, so I am just clubbing my change in this PR 😆

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,25 @@
 # Changelog
 
+## 0.9.0b1
+
+Enhancements:
+* Introduction of the user-facing `Table`, `Metadata` and `File` classes
+
+Breaking changes:
+* The operator `save_file` became `export_file`
+* The tasks `load_file`, `save_file` and `run_raw_sql` should be used with use `Table`, `Metadata` and `File` instances
```
Comment (kaxil) [inline docs/CHANGELOG.md:10]:
```suggestion
* The tasks `load_file`, `export_file` (previously `save_file`) and `run_raw_sql` should be used with use `Table`, `Metadata` and `File` instances
```

File: docs/CHANGELOG.md (hunk 3 above)
Comment (tatiana) [reply docs/CHANGELOG.md:22]: I was wondering how did that change happened :joy:

**Review by kaxil — APPROVED** [review]
1 minor comment

**Pull request merged**